modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_card_deck.yml

```diff
@@ -0,0 +1,79 @@
+id: az_paragraph_card_deck
+label: AZ Paragraph Card Deck
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: az_paragraphs_item
+  bundle: uaqs_card_deck
+  field_collection_names: 'field_uaqs_cards'
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_cards
+
+process:
+
+  field_az_cards:
+    -
+      plugin: sub_process
+      source: field_uaqs_photo
+      process:
+        delta: delta
+        media: 
+          - 
+            plugin: migration_lookup
+            source: fid
+            migration: 
+              - az_media
+    -
+      plugin: paragraphs_field_property_mapping
+      key: title
+      field_name: field_uaqs_short_title
+      value: value
+    -
+      plugin: paragraphs_field_property_mapping
+      key: body
+      field_name: field_uaqs_summary
+      value: value
+    -
+      plugin: paragraphs_field_property_mapping
+      key: body_format
+      field_name: field_uaqs_summary
```

Comment:
```suggestion
```
Is this a duplicate? According to the plugin it seems like a key of `body_format` indicates that the `field_name` isn't being used:
```
    if ($key === 'body_format' || $key === 'options') {
      $value[$key] = $value_key;
    }
    else {
      $field_name = $this->configuration['field_name'];
      $field_values = $row->getSourceProperty($field_name);
      $value[$key] = $field_values[$value['delta']][$value_key];
    }
    return $value;
    ```

Reply:
done. removed the field